src/phpFastCache/Drivers/Mongodb/Driver.php

```diff
@@ -34,11 +34,6 @@
 class Driver extends DriverAbstract
 {
     /**
-     * @var MongodbClient
-     */
-    public $instance;
```

Comment:
Can you please add the @property annotation on the class ?
This has been added to few other drivers, you can take a look.